ts/screens/profile/ProfileMainScreen.tsx

```diff
@@ -311,63 +313,79 @@ class ProfileMainScreen extends React.PureComponent<Props, State> {
       <React.Fragment>
         <SectionHeaderComponent
           sectionHeader={I18n.t("profile.main.developersSectionHeader")}
+          style={{ borderBottomWidth: 0 }}
         />
         {isPlaygroundsEnabled && (
           <>
-            <ListItemComponent
-              title={"Lollipop Playground"}
+            <ListItemNav
+              value={"Lollipop Playground"}
+              accessibilityLabel={"Lollipop Playground"}
               onPress={() =>
                 navigation.navigate(ROUTES.PROFILE_NAVIGATOR, {
                   screen: ROUTES.LOLLIPOP_PLAYGROUND
                 })
               }
             />
-            <ListItemComponent
-              title={"MyPortal Web Playground"}
+            {/* We should use FlatList or FlashList to manage
+            automatically the <Divider /> component */}
+            <Divider />
+            <ListItemNav
+              value={"MyPortal Web Playground"}
+              accessibilityLabel={"MyPortal Web Playground"}
```

Comment:
Could be useful to extract this item in a separate component instead repeat this everytime?

Reply:
Do you mean the same value for both the `value` and `accessibilityLabel` props? Note that the previous ListItem component didn't require the `accessibilityLabel' attribute, so I quickly added the same value. It could be deleted in the next releases if it's redundant for the user.